### ISSUE TITLE
Improve error messages for `Unable to find response value from` in cases where size too big

### ISSUE THREAD

**Issue by amarthadan** [api3dao/airnode#1198]
You get a cryptic error message when the API returns a response that is too large to be included on chain.

It looks like the value can't be found in the response, when in reality the response is just too big.

**Comment by bbenligiray** [comment]
Is this about https://discord.com/channels/758003776174030948/765618225144266793/976838180684824669 or something else?

**Comment by amarthadan** [comment]
It's from https://api3dao.atlassian.net/browse/AN-599 that @aquarat opened. @aquarat?

**Comment by dcroote** [comment]
AN-599 was created 09-MAR and is perhaps related to #752, for which @vponline implemented some tests here: https://github.com/api3dao/airnode/pull/818/commits/9bb995a4224460b73fce85436f0d2f6b4199c312 as part of #818.

**Comment by amarthadan** [comment]
@aquarat Can you comment on this?

**Comment by amarthadan** [comment]
Ping @aquarat ^

**Comment by aquarat** [comment]
Sorry everyone, I'm working on managing my Github notifications better.

I am investigating now.

**Comment by aquarat** [comment]
@Siegrift implemented the original response size limiting. This is circa 4th December 2021. I was tasked with testing the completed work - which I did at the time. I found that the error message in the logs didn't surface the underlying error and I noted it at the time both to Emanuel directly via Slack and apparently in the Jira issue you linked.

Looking at the code in [airnode-node](https://github.com/api3dao/airnode/blob/260faa1104ee5170c8a884ddde02702b83cb6a85/packages/airnode-node/src/api/index.ts#L238) it looks like this bug no longer exists by virtue of `processSuccessfulApiCall` wrapping the call to `extractAndEncodeResponse` inside a try/catch, where the catch returns the error as a `log` array.

So this issue can be closed, at least in terms of what it originally referenced.
